docs/how-to/LLM-Connections.md

```diff
@@ -147,7 +147,7 @@ os.environ[SOLAR_API_BASE]="https://api.upstage.ai/v1/solar"
 os.environ[SOLAR_API_KEY]="your-solar-api-key"
 ```
 
-# Free developer API key available here: https://console.upstage.ai/services/solar
+# Free developer API key available here: https://console.upstage.ai/api-keys
```

Comment:
Thanks for the PR @chosh0615, if you could remove the "#" from this line and the next one would be great.

Reply:
Hello, 

I think it was meant to be included in the codeblock. I've updated them!